code/go/0chain.net/blobbercore/blobbergrpc/proto/blobber.proto

```diff
@@ -3,6 +3,7 @@ package blobber.service.v1;
 
 option go_package = "./blobbergrpc";
 
+//import "blobbercore/blobbergrpc/proto/google/api/annotations.proto";
```

Comment:
@senpathi Use the script in scripts folder in blobbercore to generate stubs. You wont need to comment this